independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus/tooling/dockerfiles/base/src/main/docker/Dockerfile.tpl.qute.native-micro

```
@@ -19,7 +19,11 @@
 # The `{dockerfile.native-micro.from}` base image is based on UBI 9.
 # To use UBI 8, switch to `quay.io/quarkus/quarkus-micro-image:2.0`.
 ###
-FROM {dockerfile.native-micro.from}
+{#if platform.dockerfile.native-micro.from}
+FROM {platform.dockerfile.native-micro.from}
+{#else}
+FROM quay.io/quarkus/quarkus-micro-image:2.0
+{/if}
```

to have this be consistent it should use dockerfile.native-micro.from if platform version not present.
```suggestion
{#if platform.dockerfile.native-micro.from}
FROM {platform.dockerfile.native-micro.from}
{#elseif dockerfile.native-micro.from}
FROM {dockerfile.native-micro.from}
{#else}
FROM quay.io/quarkus/quarkus-micro-image:2.0
{/if}
```

But does this even work? is all of platform.* even *visible* here?